### PR TITLE
Update trip status on request on delete

### LINKED ISSUE #302: Field Trip Module: Checkmark icon persists after deleting saved itinerary
In the Field Trip Request and Field Trip Request Details views, there is an at a glance icon to let the user know if the outbound and/or inbound itineraries have been saved.  When no plans saved, it is an exclamation point and when saved, a green checkmark.

FT_SavedUnsavedIcons

However, if one has saved an itinerary, and then deletes it, the green checkmark icon persists:
Persistent checkmark after deleted itin

### PR DESCRIPTION
**Pull request by landonreed**
This fixes ibi-group/trimet-mod-otp#302. Because the status field was not being updated on delete, the trip status icon was showing a "check" even after deleting the trip.

### CODE REVIEW

File: app/controllers/FieldTrip.java [hunk 1]
```diff
@@ -549,7 +549,10 @@ public static void setRequestDate(long requestId, @As("MM/dd/yyyy") Date date) {
     public static void setRequestGroupSize(long requestId, int numStudents, int numFreeStudents, int numChaperones) {
         TrinetUser user = checkLogin();
         checkAccess(user);
-
+        System.out.println(requestId);
+        System.out.println(numStudents);
+        System.out.println(numChaperones);
+        System.out.println(numFreeStudents);
```
Comment (evansiroky) [inline app/controllers/FieldTrip.java]: Please remove these.

File: app/controllers/FieldTrip.java [hunk 2]
```diff
@@ -26,18 +20,37 @@
...
 import javax.net.ssl.HttpsURLConnection;
-import org.apache.commons.codec.digest.DigestUtils;
 
 
 public class FieldTrip extends Application {
+
+    /**
+     * Gson exclusion strategy designed to skip fields that could contribute to an overly large JSON response.
+     */
+    private static final ExclusionStrategy FIELD_TRIP_EXCLUSION_STRATEGY = new ExclusionStrategy() {
```
Comment (evansiroky) [inline app/controllers/FieldTrip.java]: This variable probably doesn't need to be in SCREAMING_SNAKE_CASE.

**Review by evansiroky — APPROVED** [review]
Good to go after addressing minor comments.

**Comment by landonreed** [comment]
@evansiroky would you be able to assign this to @binh-dam-ibigroup? He and Miles may also need to be added to this repo.

**Comment by evansiroky** [comment]
Just assigned to @binh-dam-ibigroup for final review at the request of @landonreed

File: app/controllers/FieldTrip.java (hunk 2 above)
Comment (binh-dam-ibigroup) [reply app/controllers/FieldTrip.java]: Changed case in https://github.com/opentripplanner/otp-datastore/pull/7/commits/dbf308c28c0fe69ebb1a1316cea9812af8e6a77a.

File: app/controllers/FieldTrip.java (hunk 1 above)
Comment (binh-dam-ibigroup) [reply app/controllers/FieldTrip.java]: Deleted in https://github.com/opentripplanner/otp-datastore/pull/7/commits/dbf308c28c0fe69ebb1a1316cea9812af8e6a77a.

**Review by binh-dam-ibigroup — APPROVED** [review]
LGTM

**Pull request merged**